db/seeds.rb

```diff
@@ -1,8 +1,30 @@
 # frozen_string_literal: true
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+require 'csv'
+
+ProduceItem.destroy_all
+ItemSeason.destroy_all
```

Comment:
This is fine for now, but we could use [database cleaner instead](https://github.com/DatabaseCleaner/database_cleaner). Adding the gem would also help us in testing, since it provides helpful ways for us to clean up database before and after running them.

Reply:
Where, though?

Reply:
sorry, forgot to push the latest version